draft-ietf-taps-interface.md

```diff
@@ -1242,7 +1246,7 @@ SecurityParameters.Set('pre-shared-key', key, identity)
 ~~~
 
 - Session cache management: Used to tune cache capacity, lifetime, re-use,
-and eviction policies, e.g., LRU or FIFO.may also me changed, but are implementation-specific.
+and eviction policies, e.g., LRU or FIFO.may also be changed, but are implementation-specific.
```

Comment:
```suggestion
and eviction policies, e.g., LRU or FIFO.
```

I agree with your proposed change, but this missed something more - the sentence was broken ("FIFO.may ..."). ISTM that this extra sentence is not needed here anyway.